### ISSUE TITLE
[Support]: "In Progress" events not ending, seems to prevent clips from loading?

### ISSUE THREAD

**Issue by Jonnehs** [blakeblackshear/frigate#4510]
### Describe the problem you are having

Setup has been in and working for a while, so not sure when this started happening as I only noticed it today replacing a failed camera. 

There seems to be a permanent label box detecting a car on the driveway, which is stationary and should be detected as such after 50 frames, but the detection/event continues for hours. Other events like someone walking past the car on the driveway do get detected, and there are recordings in the recording folder for them, but the Frigate UI will not play them - "The media could not be loaded, either because the server or network failed or because the format is not supported". The recordings for these events in the folder do load and play though, if manually opened.

![image](https://user-images.githubusercontent.com/11739441/204011683-add5bef2-d29b-4cc7-9e88-001137072f47.png)

![image](https://user-images.githubusercontent.com/11739441/204011562-045eb44a-4b19-4203-b67e-63efa6e33855.png)

If I go to my notification for these subsequent events on my phone it seems to try to load a long video, probably the entire recording on the ongoing detection from the car it will not stop the event for.

Future events cannot be loaded and the "in progress" event never seems to stop until the car moves out of view. 

### Version

0.11.0-3846A13

### Frigate config file

```yaml
mqtt:
  host: 10.25.1.15
  user: <MQTT>
  password: mqttpassword
birdseye:
  enabled: true
  width: 1280
  height: 720
  quality: 8
  mode: continuous
ffmpeg:
  hwaccel_args: -c:v h264_qsv
  output_args:
    record: -f segment -segment_time 60 -segment_format mp4 -reset_timestamps 1 -strftime 1 -c:v copy -c:a aac
cameras:
  Front:
    ffmpeg:
      inputs:
        - path: rtsp://frigate:frigatepassword@10.25.101.102:554/h264Preview_01_sub
          roles:
            - detect
        - path: rtsp://frigate:frigatepassword@10.25.101.102:554/h264Preview_01_main
          roles:
            - record
            - rtmp
    zones:
      frontgarden:
        coordinates: 0,205,154,204,356,178,535,159,607,116,640,138,640,480,0,480
    detect:
      width: 640
      height: 480
      fps: 7
      stationary:
        interval: 0
  Garage:
    ffmpeg:
      inputs:
        - path: rtsp://frigate:frigatepassword@10.25.101.103:554/h264Preview_01_sub
          roles:
            - detect
        - path: rtsp://frigate:frigatepassword@10.25.101.103:554/h264Preview_01_main
          roles:
            - record
            - rtmp
    detect:
      width: 640
      height: 480
      fps: 7
      stationary:
        interval: 0
  Driveway:
    ffmpeg:
      inputs:
        - path: rtsp://frigate:frigatepassword@10.25.101.101:554/h264Preview_01_sub
          roles:
            - detect
        - path: rtsp://frigate:frigatepassword@10.25.101.101:554/h264Preview_01_main
          roles:
            - record
            - rtmp
    zones:
      driveway:
        coordinates: 332,188,482,44,533,39,640,83,640,255,640,480,0,480
    detect:
      width: 640
      height: 480
      fps: 7
      stationary:
        interval: 0
motion:
  threshold: 40
  contour_area: 40
snapshots:
  enabled: true
  bounding_box: true
  retain:
    default: 60
record:
  enabled: True
  retain:
    days: 21
    mode: motion
  events:
    pre_capture: 20
    retain:
      default: 21
      mode: active_objects
      objects:
        person: 60
        car: 60
        motorcycle: 60
        dog: 10
        cat: 10
live:
  height: 720
  quality: 5
objects:
  track:
    - person
    - car
    - motorcycle
    - cat
    - dog
    - bird
detectors:
  coral:
    type: edgetpu
    device: usb
```

### Relevant log output

```shell
[2022-11-25 14:44:03] frigate.app                    INFO    : Starting Frigate (0.11.0-3846a13),
[2022-11-25 15:01:38] ws4py                          INFO    : Terminating websocket [Local => 127.0.0.1:8082 | Remote => 127.0.0.1:41080],
[2022-11-25 14:44:03] ws4py                          INFO    : Using epoll,
Starting migrations,
[2022-11-25 14:44:03] peewee_migrate                 INFO    : Starting migrations,
There is nothing to migrate,
[2022-11-25 14:44:03] peewee_migrate                 INFO    : There is nothing to migrate,
[2022-11-25 14:44:03] detector.coral                 INFO    : Starting detection process: 224,
[2022-11-25 14:44:03] frigate.app                    INFO    : Output process started: 225,
[2022-11-25 14:44:03] frigate.app                    INFO    : Camera processor started for Front: 229,
[2022-11-25 14:44:03] frigate.app                    INFO    : Camera processor started for Garage: 233,
[2022-11-25 14:44:03] frigate.app                    INFO    : Camera processor started for Driveway: 236,
[2022-11-25 14:44:03] frigate.app                    INFO    : Capture process started for Front: 237,
[2022-11-25 14:44:03] frigate.app                    INFO    : Capture process started for Driveway: 243,
[2022-11-25 14:44:03] frigate.app                    INFO    : Capture process started for Garage: 239,
[2022-11-25 14:44:03] frigate.edgetpu                INFO    : Attempting to load TPU as usb,
[2022-11-25 14:44:06] frigate.edgetpu                INFO    : TPU found,
[2022-11-25 14:44:23] ws4py                          INFO    : Managing websocket [Local => 127.0.0.1:8082 | Remote => 127.0.0.1:41072],
[2022-11-25 14:46:23] ws4py                          INFO    : Managing websocket [Local => 127.0.0.1:8082 | Remote => 127.0.0.1:41074],
[2022-11-25 14:44:25] ws4py                          INFO    : Terminating websocket [Local => 127.0.0.1:8082 | Remote => 127.0.0.1:41072],
[2022-11-25 14:46:24] ws4py                          INFO    : Terminating websocket [Local => 127.0.0.1:8082 | Remote => 127.0.0.1:41074],
[2022-11-25 14:46:59] ws4py                          INFO    : Managing websocket [Local => 127.0.0.1:5002 | Remote => 127.0.0.1:57414],
[2022-11-25 14:47:00] ws4py                          INFO    : Terminating websocket [Local => 127.0.0.1:5002 | Remote => 127.0.0.1:57414],
[2022-11-25 14:47:00] ws4py                          INFO    : Managing websocket [Local => 127.0.0.1:5002 | Remote => 127.0.0.1:57416],
[2022-11-25 14:47:00] ws4py                          INFO    : Managing websocket [Local => 127.0.0.1:5002 | Remote => 127.0.0.1:57418],
[2022-11-25 14:47:00] ws4py                          INFO    : Terminating websocket [Local => 127.0.0.1:5002 | Remote => 127.0.0.1:57416],
[2022-11-25 14:50:44] ws4py                          INFO    : Managing websocket [Local => 127.0.0.1:8082 | Remote => 127.0.0.1:41076],
[2022-11-25 14:51:03] ws4py                          INFO    : Managing websocket [Local => 127.0.0.1:8082 | Remote => 127.0.0.1:41078],
[2022-11-25 14:50:48] ws4py                          INFO    : Terminating websocket [Local => 127.0.0.1:8082 | Remote => 127.0.0.1:41076],
[2022-11-25 14:51:05] ws4py                          INFO    : Terminating websocket [Local => 127.0.0.1:8082 | Remote => 127.0.0.1:41078],
[2022-11-25 14:55:08] ws4py                          INFO    : Terminating websocket [Local => 127.0.0.1:5002 | Remote => 127.0.0.1:57418],
[2022-11-25 14:55:08] ws4py                          INFO    : Managing websocket [Local => 127.0.0.1:5002 | Remote => 127.0.0.1:57420],
[2022-11-25 15:01:35] ws4py                          INFO    : Managing websocket [Local => 127.0.0.1:8082 | Remote => 127.0.0.1:41080]
```

### FFprobe output from your camera

```shell
ffprobe version 4.4.1-full_build-www.gyan.dev Copyright (c) 2007-2021 the FFmpeg developers
  built with gcc 11.2.0 (Rev1, Built by MSYS2 project)
  configuration: --enable-gpl --enable-version3 --enable-static --disable-w32threads --disable-autodetect --enable-fontconfig --enable-iconv --enable-gnutls --enable-libxml2 --enable-gmp --enable-lzma --enable-libsnappy --enable-zlib --enable-librist --enable-libsrt --enable-libssh --enable-libzmq --enable-avisynth --enable-libbluray --enable-libcaca --enable-sdl2 --enable-libdav1d --enable-libzvbi --enable-librav1e --enable-libsvtav1 --enable-libwebp --enable-libx264 --enable-libx265 --enable-libxvid --enable-libaom --enable-libopenjpeg --enable-libvpx --enable-libass --enable-frei0r --enable-libfreetype --enable-libfribidi --enable-libvidstab --enable-libvmaf --enable-libzimg --enable-amf --enable-cuda-llvm --enable-cuvid --enable-ffnvcodec --enable-nvdec --enable-nvenc --enable-d3d11va --enable-dxva2 --enable-libmfx --enable-libglslang --enable-vulkan --enable-opencl --enable-libcdio --enable-libgme --enable-libmodplug --enable-libopenmpt --enable-libopencore-amrwb --enable-libmp3lame --enable-libshine --enable-libtheora --enable-libtwolame --enable-libvo-amrwbenc --enable-libilbc --enable-libgsm --enable-libopencore-amrnb --enable-libopus --enable-libspeex --enable-libvorbis --enable-ladspa --enable-libbs2b --enable-libflite --enable-libmysofa --enable-librubberband --enable-libsoxr --enable-chromaprint
  libavutil      56. 70.100 / 56. 70.100
  libavcodec     58.134.100 / 58.134.100
  libavformat    58. 76.100 / 58. 76.100
  libavdevice    58. 13.100 / 58. 13.100
  libavfilter     7.110.100 /  7.110.100
  libswscale      5.  9.100 /  5.  9.100
  libswresample   3.  9.100 /  3.  9.100
  libpostproc    55.  9.100 / 55.  9.100
Input #0, rtsp, from 'rtsp://frigate:frigatepassword@10.25.101.101:554/h264Preview_01_sub':
  Metadata:
    title           : Session streamed by "preview"
    comment         : h264Preview_01_sub
  Duration: N/A, start: 0.000188, bitrate: N/A
  Stream #0:0: Video: h264 (High), yuv420p(progressive), 640x480, 29.97 tbr, 90k tbn, 180k tbc
  Stream #0:1: Audio: aac (LC), 16000 Hz, mono, fltp
```

### Frigate stats

```json
{"Driveway":{"camera_fps":7.1,"capture_pid":243,"detection_fps":0.0,"pid":236,"process_fps":7.1,"skipped_fps":0.0},"Front":{"camera_fps":7.1,"capture_pid":237,"detection_fps":1.2,"pid":229,"process_fps":7.1,"skipped_fps":0.0},"Garage":{"camera_fps":7.1,"capture_pid":239,"detection_fps":0.0,"pid":233,"process_fps":7.1,"skipped_fps":0.0},"detection_fps":1.2,"detectors":{"coral":{"detection_start":0.0,"inference_speed":9.84,"pid":224}},"service":{"latest_version":"0.11.1","storage":{"/dev/shm":{"free":1070.9,"mount_type":"tmpfs","total":1073.7,"used":2.9},"/media/frigate/clips":{"free":742562.2,"mount_type":"ext4","total":980667.0,"used":197116.7},"/media/frigate/recordings":{"free":742562.2,"mount_type":"ext4","total":980667.0,"used":197116.7},"/tmp/cache":{"free":742562.2,"mount_type":"overlay","total":980667.0,"used":197116.7}},"temperatures":{},"uptime":1470,"version":"0.11.0-3846a13"}}
```

### Operating system

Other Linux

### Install method

Docker Compose

### Coral version

USB

### Network connection

Wired

### Camera make and model

Reolink 510A

**Comment by Jonnehs** [comment]
I suppose, for clarity, there is a risk here that I am conflating two different issues. Detections and events still seem to happen on my "Driveway" camera, but no clips can be played from the frigate UI or the home assistant URLs for that camera. They can be played from the recordings folder manually though.

The "In progress" events that never seem to stop tracking stationary objects could be another issue, but its the same Driveway camera also doing that so, to my mind they might be related..

**Comment by NickM-27** [comment]
These things aren't related, also the in progress events that stay is the expected behavior and not a bug. The event will last as long as the object is visible. 

As far as the media could not be loaded, having the browser logs might be helpful otherwise it might be an issue related to a corrupt segment that was inadvertently saved.

**Comment by Jonnehs** [comment]
Ok, cool. The clips it gives me are corrupted, but im not sure why or how as all the source video seems to work, and its only for this one camera. I downloaded the clip it tries to send me for notifications and it seems pretty broken

![image](https://user-images.githubusercontent.com/11739441/204020349-413dc6c5-b478-42fa-8d46-a32002ca1e8f.png)

the browser says its 12 minutes long, the properties say 16:04 and really it should just be less than 10 seconds. This camera did blow up and I had to replace it, water damage - got inside its poe gromit, but the replacement camera works fine in every way except its event clips. The snapshot screenshots are all good, all work, just the clip recordings are borked.

**Comment by Jonnehs** [comment]
Console looks like this, seems to be getting a 503 from frigate for the clip URL?

![image](https://user-images.githubusercontent.com/11739441/204023600-e6e826d9-26ff-42f5-a41b-f9fc2b531381.png)

http://10.25.1.16:5000//vod/event/1669391109.830802-uufzln/master.m3u8

gives 503 Service Temporarily Unavailable

![image](https://user-images.githubusercontent.com/11739441/204023701-a45c4d6b-c43f-43c5-8c0a-d5adf0d910e3.png)

**Comment by Jonnehs** [comment]
I deleted my frigate.db  frigate.db-shm  frigate.db-wal and its working fine again on restart. Will it automatically clean up the old recordings which aren't in the DB, or should I delete those manually too?

Do you want the broken DB for any investigation into what caused this?

I'm happy enough to reset and get it working again so feel free to close this.

**Comment by NickM-27** [comment]
Thanks, you will need to delete those old recording files. And we already know what this issue is and have fixed it for 0.12.

**Comment by conorlap** [comment]
> Thanks, you will need to delete those old recording files. And we already know what this issue is and have fixed it for 0.12.

Hey @NickM-27 is there anything us users can do locally to fix this in 0.11.1 before 0.12 is released? I have to delete my entire database every few days otherwise I can't playback any recordings after a corruption.
